closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -987,9 +1240,124 @@ BIGNUM* convertJavaBItoBN(unsigned char* in, int len) {
             }
         }
     }
-    BIGNUM* bn = BN_bin2bn(in, len, NULL);
+    BIGNUM* bn = (*OSSL_BN_bin2bn)(in, len, NULL);
     if (bn != NULL) {
-        BN_set_negative(bn, neg);
+        (*OSSL_BN_set_negative)(bn, neg);
     }
     return bn;
 }
+
+typedef struct rsa_st102 {
+/*
+      * The first parameter is used to pickup errors where this is passed
+      * instead of aEVP_PKEY, it is set to 0
```

Comment:
indentation?

Reply:
This is very pedantic, but it has been done.